client/asset/btc/spv_wrapper.go

```diff
@@ -1157,8 +1158,54 @@ func (w *spvWallet) moveWalletData(backupDir string) error {
 	if err != nil {
 		return err
 	}
-	backupFolder := filepath.Join(backupDir, timeString)
-	return os.Rename(w.dir, backupFolder)
+
+	backupFolder := filepath.Join(backupDir, w.chainParams.Name, timeString)
+	// Copy wallet logs first. Even if there is an error, wallet files are
+	// still intact.
+	backupLogDir := filepath.Join(backupFolder, logDirName)
+	walletLogDir := filepath.Dir(w.logFilePath())
+	if err := copyDir(walletLogDir, backupLogDir); err != nil {
+		return err
+	}
+
+	walletBackupDir := filepath.Join(backupFolder, spvDir)
+	if err := os.Rename(w.dir, walletBackupDir); err != nil {
+		return err
+	}
+	return nil
+}
+
+// copyDir recursively copies the directories and files in source directory to
+// destination directory without preserving the original file permissions.
+func copyDir(srcDir, dstDir string) error {
+	dirFiles, err := os.ReadDir(srcDir)
+	if err != nil {
+		return err
+	}
+
+	if err := os.MkdirAll(dstDir, 0744); err != nil {
+		return err
+	}
```

Comment:
The docs for `copyDir` should also say `The destination folder must not exist.` if that's gonna be the restriction.

Reply:
I think it should be a restriction.